client/src/pages/UserAdmin.js

```diff
@@ -47,9 +47,9 @@ const UserAdmin = () => {
     fetchProjects();
   }, [fetchUsers, fetchProjects]);
 
-  const backToSearch = () => {
-    setUserToEdit({});
-  };
+  // const backToSearch = () => {
```

Comment:
Pretty sure changes to this file were a mistake, please undo changes to `UserAdmin.js`.

Reply:
@Spiteless  fixed